Makefile

```diff
@@ -53,7 +54,7 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-GOLANGCI_LINT = $(GOBIN)/golangci-lint
+GOLANGCI_LINT = $(BIN_DIR)/golangci-lint
```

Comment:
We can also get rid of [L51](https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/399/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R51). Right?

```
# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
ifeq (,$(shell go env GOBIN))
GOBIN=$(shell go env GOPATH)/bin
else
GOBIN=$(shell go env GOBIN)
endif
```

Reply:
yes this logic is no longer needed in Makefile. will remove